docs/cli.md

```diff
@@ -139,6 +139,11 @@ The environment variables listed below take precedence over the DataHub CLI conf
 - `ACTIONS_VERSION` (default `head`) - Set to a specific version to run quickstart with that image tag of `datahub-actions` container.
 - `DATAHUB_ACTIONS_IMAGE` (default `acryldata/datahub-actions`) - Set to `-slim` to run a slimmer actions container without pyspark/deequ features.
 
+For images `acryldata/datahub-ingestion` or `acryldata/datahub-ingestion-slim`
+
+- `ACTIONS_EXTRA_PACKAGES` (default ``) - Install the extra python packages prior to running the main executable.
+- `ACTIONS_CONFIG` (default ``) - Use the specified actions configuration file url (yaml format).
```

Comment:
For end users, we should recommend using the actual `acryldata/datahub-actions` images instead of these right? If that's the case, should we even document these args here - my worry is that they might cause more confusion

If we still want to keep these docs here, we should make it clear that if either is set, the container runs in "actions mode", but by default it runs in "datahub mode"

Reply:
I can just remove these, since they might be confusing. No problem.